docs/library/graphing/bar_chart.md

```diff
@@ -149,6 +163,35 @@ range_bar_chart = """rx.bar_chart(
                 rx.x_axis(data_key="day"), 
                 rx.y_axis(),
                 data=range_data)"""
+
+bar_chart_example_with_state = """rx.bar_chart(
+            rx.bar(
+                data_key="uv",
+                stroke="#8884d8",
+                fill="#8884d8",
+                type_="natural",
+                on_click=BarState.randomize_data,
+
+            ),
+            rx.bar(
+                data_key="pv",
+                stroke="#82ca9d", 
+                fill="#82ca9d",
+                type_="natural",
+            ),
+            rx.x_axis(
+                data_key="name",
+            ),
+            rx.y_axis(), 
+            rx.legend(),
+            rx.cartesian_grid(
+                stroke_dasharray="3 3",
+            ),
+            data=BarState.data,
+            width=1000,
```

Comment:
Same here avoid setting an explicit width height or choose to set an explicit height but width should be 100%

Reply:
changed